server/rpc_test.go

```diff
@@ -561,11 +572,17 @@ func defaultPayment(t *testing.T) *net.Payment {
 
 func defaultPaymentWithTickets(t *testing.T, senderParams []*net.TicketSenderParams) *net.Payment {
 	sender := pm.RandBytes(123)
-
+	oinfo := &net.OrchestratorInfo{
```

Comment:
Doesn't look like we need to create `*net.OrchestratorInfo` here since we just need to provide `*net.PriceInfo` for the `ExpectedPrice` field when creating `*net.Payment`.

Reply:
Ah yes true, I was being lazy and copy pasted that from the `genPayment` test.

Fixed in https://github.com/livepeer/go-livepeer/pull/958/commits/ee0955b26e5234bee08257d1a055f607be90a51a